EditorDelegates/EditorDelegates.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+namespace Unity.Labs.SuperScience
+{
+    /// <summary>
+    /// This class contains functionality provided by code in the Editor assembly
```

Comment:
Seems more like this class is for routing editor functionality than containing it?